gemfiles/mime_types_2.latest.gemfile

```diff
@@ -1,12 +0,0 @@
-# This file was generated by Appraisal
-
-source "https://rubygems.org"
-
-gem "jruby-openssl", :platforms => :jruby
-gem "mime-types", "~> 2.3"
-
-group :local_development, :test do
-  gem "appraisal", "~> 1.0"
-end
-
-gemspec :path => "../"
```

Comment:
It was removed from the [Appraisals file](https://github.com/mikel/mail/commit/d4ffe72b0b2b0c6c8a49130bfe8e792d0542ac0e) which is what this is auto-generated from. I think it is assumed that 2.99 will always be the [latest 2.x ](https://rubygems.org/gems/mime-types/versions/2.99)